wolox-books/src/app/components/user/user.service.ts

```diff
@@ -29,4 +30,9 @@ export class UserService {
     json[b] = json[a];
     delete json[a];
   }
+
+  public loggedIn(){
+    return !!this.localStorage.getValue('accessToken');
```

Comment:
I think there's no need for the falsy. If it's `undefined` it will be `false` and if not, `true`